src/Altinn.Platform/Altinn.Platform.PDF/pom.xml

```diff
@@ -32,15 +32,15 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>3.4.4</version>
+    <version>3.4.5</version>
   </parent>
   <dependencies>
     <dependency>
       <groupId>org.apache.pdfbox</groupId>
       <artifactId>pdfbox</artifactId>
       <!-- Updating to 3.0.0 will not be prioritized since this component will be
         replaced by new pdf generator -->
-      <version>2.0.33</version>
+      <version>2.0.34</version>
```

Comment:
Unsure about this as the task says `do not update version on <groupId>org.apache.pdfbox</groupId>`, but this is a patch version and not a major

Reply:
I want us to update patch versions. We have in the past.